### DISCUSSION TITLE
Is it possible to use KenLm language models or other with ContextNet?

### DISCUSSION THREAD

**Discussion by Slava715** [NVIDIA-NeMo/NeMo#2879] [Q&A]
Is it possible to use KenLm language models or other with ContextNet?

**Comment by titu1994** [comment]
Not yet. We are looking into it.

**Reply by Slava715** [reply]
Thank you very much! This should significantly improve the quality of the model.

**Reply by titu1994** [reply]
Not really. RNNT have inherent language model in them, and are Autoregressive. The benefit of LM rescoring will be quite minor
